components/ide/code/base.Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ubuntu:18.04
 
 # we use latest major version of Node.js distributed VS Code. (see about dialog in your local VS Code)
 # ideallay we should use exact version, but it has criticla bugs in regards to grpc over http2 streams
-ARG NODE_VERSION=14.17.6
```

Comment:
it actually does not matter, new VS Code always bring own node, not something specified here